src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1777,6 +1777,16 @@ internal enum ErrorCode
         ERR_ExpressionTreeContainsFromEndIndexExpression = 8791,
         ERR_ExpressionTreeContainsRangeExpression = 8792,
 
+        ERR_PartialMethodWithNonVoidReturnMustHaveImplementation = 9050,
```

Comment:
>9050 [](start = 67, length = 4)

Consider using one of the gaps above (for instance, 8755-), especially if some of these errors will be collapsed.

Reply:
Fixed